hack/local-up-cluster.sh

```diff
@@ -202,6 +202,18 @@ EXPERIMENTAL_CRI=${EXPERIMENTAL_CRI:-"false"}
 # Ensure CERT_DIR is created for auto-generated crt/key and kubeconfig
 mkdir -p "${CERT_DIR}" &>/dev/null || sudo mkdir -p "${CERT_DIR}"
 CONTROLPLANE_SUDO=$(test -w "${CERT_DIR}" || echo "sudo -E")
+# If on an selinux system, need to set the correct context so cfssl container can access it
+if command -v getenforce &>/dev/null; then
+    if [[ "$(getenforce)" != "Disabled" ]]; then
+        # Try newer container_file_t type first
+        if ! sudo chcon -R -t container_file_t "${CERT_DIR}"; then
```

Comment:
We run the risk here of aliasing `container_file_t` not existing with actual failures to `chcon` the directory. If we have / can install the SELinux tools console and use `seinfo -t` to list types available, we could make this check better. @sttts or @pmorie could comment on if that is necessary.